tools/reindex-index-data

```diff
@@ -38,8 +52,11 @@ def reindex(index, doc_types):
     # Reindex with the same data
     for doc_type, records in doctype_records.items():
         print('Indexing', len(records), index + '/' + doc_type, 'records.')
-        for r in records:
-            conn.index(body=r, index=index, doc_type=doc_type)
+        try:
+            elasticsearch.helpers.bulk(conn, index_iterator(records, index, doc_type))
+        except ElasticsearchException as e:
+            print('Indexing failed: {}'.format(e), file=sys.stderr)
+            continue
```

Comment:
Maybe also print the index/doctype/record that failed so it can be manually retried if needed?